scalableminds/webknossos
# Prevent errors thrown by keyboard shortcuts used int the 3D viewport

This PR should fix all error currently thrown by shortcuts triggered in the 3D viewport that are not meant for the 3D viewport.
Examples for these short cuts are: d, f, space, v.

### URL of deployed dev instance (used for testing):
- https://lesserrorsindviewport.webknossos.xyz

### Steps to test:
- Open a hybrid tracing in wk.
- Use the shortcuts
  - d
  - f
  - space
  - space + shift
In the 3d viewport. They should do nothing and should not throw an error.
- Test the listed shortcuts in one of the other viewports and make sure they do work.
- Please also test whether other shortcuts trigger an error or unwanted side effect when used in the 3d viewport. I tried to catch every keyboard shortcut.

### Issues:
- fixes #5975

------
(Please delete unneeded items, merge only when none are left open)
